components/dashboard/script.js

```diff
@@ -0,0 +1,37 @@
+const fs = require('fs');
+
+const vars = {
+    REACT_APP_HOSTNAME : 'Gitpod',
```

Comment:
<details>
<summary>:warning: <strong>Formatting related to spacing is inconsistent.</strong> (click for details)</summary>

A possible improvement would be to apply the current language Coding Conventions (or project specific style) to all files and fix any possible spacing issue before committing to maintain code style consistency.  
  
This change would ensure that the code is consistently clean, thus, improving readability, making variable and block scopes easier to understand.

**[More details](https://confluence.devfactory.com/x/bHvlFQ)**

## The improvement suggestion in this case
```js  
REACT_APP_HOSTNAME: 'Gitpod',
```  
> NOTE: Consider applying the structure of this solution in other parts of the code, if there is another occurrence(s).
</details>

[//]: # (<a id="771456ab-efcc-4217-b7c7-26b9d3be7cc7" data-finding-id="367360876" href="none" codereview-uuid-reserved-tag data-business-id="05d37d16-d6de-46e5-9cc8-3f767198a252" data-analyzer="js" />)